### PR TITLE
New BashCommands CheatSheet

### PR DESCRIPTION
**Pull request by SwathiSBhat**
## Type of Change

- [X ] New Instant Answer
  - [X ] Cheat Sheet
- [ X] Improvement
  - [ ] Bug fix
  - [X ] Enhancement
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.)
## People to notify

Contact me(@SwathiSBhat) at my Github profile for any  suggestions or improvements.
---

IA Page: https://duck.co/ia/view/bashcommands_cheatsheet

Forum Topic:

### CODE REVIEW

**Comment by daxtheduck** [comment]
@SwathiSBhat Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by jophab** [comment]
@SwathiSBhat Please remove the second file from this PR.

File: share/goodie/cheat_sheets/json/bash1.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "id": "bash1_cheat_sheet",
+    "name": "Bash1",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/bash1.json]: Why is this 'Bash1'?

File: share/goodie/cheat_sheets/json/bash1.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "description": "Displays a list of commonly used bash commands",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/bash1.json]: "A list of commonly used bash commands" would be enough

File: share/goodie/cheat_sheets/json/bash1.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            {
+                "key": "less",
+                "val": "Display output one screen at a time"
+            },
+            {
+                "key": "uniq",
+                "val": "Uniquify files"
+            }
+        ],
+        "Network configuration and troubleshooting": [
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/bash1.json]: You missed this in section order.
Also make it "Network Configuration and Troubleshooting"

File: share/goodie/cheat_sheets/json/bash1.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    },
+
+    "aliases": [
+        "bash commands", "bourne shell commands", "common bash commands"
+    ],
+
+    "template_type": "code",
+
+    "section_order": [
+        "File operations",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/bash1.json]: Lets make it "File Operations"

**Comment by jophab** [comment]
@SwathiSBhat The json in invalid. 
Use jsonlint to find the error and correct it.

**Comment by sahildua2305** [comment]
@SwathiSBhat you may also use our own interactive IA for validating and checking for errors in your json file. https://duckduckgo.com/?q=json+validate&t=h_&ia=answer

@jophab 😉

**Comment by jophab** [comment]
@sahildua2305 I didn't knew about that one man :wink: 
Good work :+1: :smiley:

**Comment by jophab** [comment]
@sahildua2305 I didn't knew about that one man :wink: 
Good one,  A very useful IA :+1: :smiley:

**Comment by amarlearning** [comment]
@SwathiSBhat  change the file name to `bash.json` instead of `bash1.json`! Rest looks nice :grin:

**Comment by SwathiSBhat** [comment]
@amarlearning bash.json already exists so I renamed it to Bash.json

**Comment by SwathiSBhat** [comment]
@amarlearning bash.json already exists so I renamed it to Bash.json :)

**Comment by amarlearning** [comment]
@SwathiSBhat As you know, there is already a cheat sheet named `Bash`, I would suggest you close this PR. You should make changes in the existing `bash cheat sheet`  and then open a PR.

**Pull request closed without merging**